tasks/docs.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+from invoke import task
+
+from .util import PROJECT_DIR, print_and_run
+
+
+@task()
+def build(_ctx):
```

Comment:
a `watch` task would also be nice for when docs are being actively worked on.

Reply:
:100:

Reply:
Added this and a `clean` task as well.